setup.py

```diff
@@ -1,53 +1,27 @@
+#!/usr/bin/env python
+
 # this script allows us to publish sureal in PyPi and the use it using 'pip install sureal'
 # run 'python setup.py sdist bdist_wheel' to create the distribution files within sureal/python/src/
 
-import os
-import setuptools
-from numpy.distutils.core import setup
```

Comment:
`numpy.distutils` seems to be mostly interesting for `Extensions`, it seems that here we don't need it (it's also unclear how to properly get it from their doc, they seem to assume that one has numpy basically installed globally...)